python/ray/util/client/server/dataservicer.py

```diff
@@ -206,7 +205,8 @@ def Datapath(self, request_iterator, context):
                 # Connection isn't recoverable, skip cleanup
                 cleanup_requested = True
         finally:
-            logger.debug(f"Lost data connection from client {client_id}")
+            logger.debug(f"Stream is broken with client {client_id}")
+            self.basic_service.release_all(client_id)
```

Comment:
the release_all here can be removed (it now happens farther down at what is currently line 241, after the grace period has passed). Guessing this was a remnant of resolving merge conflicts

Reply:
Removed, thanks!